doc/DeveloperGuide.md

```diff
@@ -88,6 +88,27 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename tag
+
+**MSS**
+
+1. User requests to rename a tag
+2. AddressBook shows the previous name and new name and asks for confirmation
```

Comment:
1 scenario per point.
1. Address book asks for confirmation can be a diff point